test_export.py

```
@@ -0,0 +1,161 @@
+
+        print(f"✅ PDF generated successfully!")
+        print(f"   File: {output_file}")
+        print(f"   Size: {len(pdf_bytes):,} bytes ({len(pdf_bytes)/1024:.2f} KB)")
+        print(f"\n📊 Report Statistics:")
+        print(f"   - Risk Score: {resultado.score_risco}/100")
+        print(f"   - Alerts: {len(resultado.alertas)}")
+        print(f"   - Status: {'Safe' if resultado.seguro else 'Attention Required'}")
+
+        return True
+
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)